toast-tk-runtime/src/main/java/io/toast/tk/runtime/TestRunner.java

```diff
@@ -86,19 +96,60 @@ private void enrichTestBlockResults(ITestPage testPage, TestBlock testBlock) {
 		testBlock.setTestFailureNumber(nbBlockFailures);
 
 	}
+	
+	public void kill() {
+		this.interupted = true;
+		if(blockRunner != null) {
+			if(blockRunner instanceof TestBlockRunner) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)